app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+
+from app.cafe import Cafe
+
+from app.errors import (VaccineError, NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    errors = []
+    masks_to_buy = 0
+
+    for visitor in friends:
+
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError as e:
+            errors.append(e)
```

Comment:
There is no need to allocate `errors` variable and append an exception to it because in case the `VaccineError` will be raised, the next step would be a `return` statement without further function execution. Remove this line, the `errors` variable, and other logic related to it.

Reply:
i don`t understand